src/assets/i18n/de_DE.json

```diff
@@ -650,6 +650,16 @@
   "checkout.credit_card.user.lastname.missing.error": "Bitte geben Sie den Nachnamen des Karteninhabers an.",
   "checkout.credit_card.user.name.missing.error": "Bitte geben Sie den Namen des Karteninhabers an.",
   "checkout.credit_card.validityTime.error.notFound": "Die maximale Gültigkeitsdauer der Kartenprüfnummer wurde nicht gefunden.",
+  "checkout.desired_delivery_date.apply.button.label": "Übernehmen",
+  "checkout.desired_delivery_date.error.date": "Das Datum ist ungültig. Bitte verwenden Sie das korrekte Format (TT.MM.JJJJ).",
```

Comment:
Please check if the date picker plugin has localized date formats. Else the given format in our messages has to be changed accordingly.